docs/documentation.rst

```diff
@@ -0,0 +1,8 @@
+Explain PRIMO in One Page
```

Comment:
I would change this to either "PRIMO in One Page" or "PRIMO explained in one page" or "Explanation of PRIMO in one page" (how you capitalize is not important to me...)

Reply:
It shows up on the left hand navigation bar, so maybe briefer is better?
![Screenshot 2024-10-09 112016](https://github.com/user-attachments/assets/04258d60-7260-4b59-aca8-f9d301ab0aba)

Reply:
Changed it to "PRIMO Explained in One Page". I thought about the idea of using "PRIMO in One Page" but I was afraid that it may sound like the PRIMO tool (code) can fit in one page. Thus, I decided to add the work "Explain" in the title.